package.json

```diff
@@ -8,10 +8,10 @@
   },
   "scripts": {
     "build": "tsc",
-    "coverage": "istanbul cover _mocha",
-    "coveralls": "npm run coverage && coveralls <coverage/lcov.info",
+    "coverage": "nyc mocha --reporter spec --require ts-node/register 'test/index.ts'",
+    "coveralls": "npm run coverage && nyc report --reporter=text-lcov | coveralls",
     "lint": "standard",
-    "test": "istanbul test mocha -- --reporter spec",
+    "test": "mocha --reporter spec --require ts-node/register 'test/index.ts'",
```

Comment:
Cannot make it work with `'test/index.ts'` on windows cmd. I need to remove the quotes.